### PR TITLE
Add AnanjayGurjar.json

### PR DESCRIPTION
**Pull request by AnanjayGurjar**
Please, Add my profile to the main project. Open for suggestions on how to make it better.
P.S. It's my first contribution to any open source project.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1622"></a>

### CODE REVIEW

File: public/data/AnanjayGurjar.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
```
Comment (Panquesito7) [inline public/data/AnanjayGurjar.json:1]: There's an extra line here.
```suggestion
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

File: public/data/AnanjayGurjar.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        "color": "#0a0a23",
+        "description": "Began working as a core member for RIT chapter of Google Developer Student Club, a non-profit organisation that aims to enhances the coding culture in college by organizing various events",
+        "url": "https://github.com/DSC-RIT"
+      },
+      {
+        "title": "Started as Android Intern at Meduto",
+        "date": "Jan 2022",
+        "icon": "android",
+        "color": "#3a3240",
+        "description": "Worked with Meduto as Android Developer Intern where I worked on the ",
```
Comment (kumarsonsoff3) [inline public/data/AnanjayGurjar.json:37]: This sentence seems to be incomplete, please complete it or modify it! 🛠

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**